src/vim9execute.c

```diff
@@ -3813,13 +3813,21 @@ exec_instructions(ectx_T *ectx)
 	    case ISN_STORE:
 		--ectx->ec_stack.ga_len;
 		tv = STACK_TV_VAR(iptr->isn_arg.number);
-		if (STACK_TV_BOT(0)->v_type == VAR_TYPEALIAS)
+		// DON'T THINK THIS EVER TRIGGERS, GETS CAUGHT IN COMPILATION?
```

Comment:
From the Codecov results, I do see the below error message is covered.  You can try commenting out this code and run the tests and see which test fails.

Reply:
Turns out this is catching
```
var x = eval("A")
```
in a function.